cmd/state/internal/cmdtree/cmdtree.go

```diff
@@ -290,6 +291,13 @@ func newStateCommand(globals *globalOptions, prime *primer.Values) *captain.Comm
 				Description: locale.T("flag_state_version_description"),
 				Value:       &opts.Version,
 			},
+			{
+				Name:        "help",
+				Description: locale.Tl("flag_help", "Help for this command"),
+				Shorthand:   "h",
+				Persist:     true,
```

Comment:
Will this "override" the output of the help descriptions/messages of subcommands?

Reply:
Yes, this is a single, universal message for all of our commands and subcommands.